tests/test_main.py

```diff
@@ -24,10 +24,20 @@
 from .conftest import log_check
 
 
+@pytest.fixture
+def data_instance():
+    """Creates an emmpty data instance."""
+    from virtual_rainforest.core.data import Data
+    from virtual_rainforest.core.grid import Grid
+
+    grid = Grid()
+    return Data(grid)
```

Comment:
Given that this fixture is identical to the one in `test_soil_model`, I would add it to a `conftest.py` file in the tests root directory.